tools/ocean_data_view_manager/tool_odv.xml

```diff
@@ -0,0 +1,163 @@
+    <description>tool for qualificiation and validation</description>
+    <macros>
+        <token name="@VERSION@">0.2</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <container type="docker">pokapok/qcv-odv-tool:@VERSION@</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        export HOME=\$PWD &&
+        mkdir -p /runtime/config &&
+        mkdir -p '/runtime/data-in-raw/' &&
+        mkdir -p '/runtime/data-in-ref/' &&
+        mkdir -p '/runtime/data-out/' &&
+
+        cp '$input_raw' '/runtime/data-in-raw/${input_raw.element_identifier}' &&
+        #if $input_ref and str($input_ref).strip() not in ["", "None"]:            
+            cp '$input_ref' '/runtime/data-in-ref/${input_ref.element_identifier}' &&
+        #end if
+
+        #if $config.type == 'yes':
+            ## Define output YAML file
+            touch /runtime/config/config_file.yaml
+
+            ## Write YAML content
+            '/runtime/config/config_file.yaml' << echo"
+            default :
+            ## mandatory information
+                file_name_list_raw :
+                    '\$config.file_name_list_raw.element_identifier'    
+            ## optional
+                operator : '\$config.operator'
+            QV :
+                file_name_list_ref : 
+                    '\$config.file_name_list_ref.element_identifier'
+                odv_collection :
+                    name : odv_collection.txt
+                    qc_convention : '\$config.qc_convention'
+                tool_option : 
+                    subsetting : '\$config.subsetting'
+                    plt : '\$config.plt'            
+            " &&
+        #end if 
+
+        /app/launchers/start-app.sh '$mode' &&
+        #if $mode == 'R_APP_QV':
+            cp '/runtime/data-out/odv_collection.txt' $output &&
+        #else:
+            cp '/runtime/data-out/demo_odv_collection.txt' $output &&
+        #end if
+        cp -r '/runtime/data-out/' './'
+    ]]></command>
+    <inputs>
+        <param name="input_raw" type="data" format="netcdf" label="Input raw data"/>
+        <param name="input_ref" type="data" format="netcdf" optional="true" label="Input ref data"/>
+        <param name="mode" type="select" label="Select in which mode you want to run the tool">
+            <option value="R_APP_QV">QV that creates a odv collection with raw and ref files to qualify</option>
+            <option value="DEMO">Valide with ODV demo that performes the demo</option>
+        </param>
+        <conditional name="config">
+            <param name="type" type="select" label="Select if you are have your own configuration file or not.">   
+                <option value="no">No, I don't have my a configuration file, I use the default one</option>
+                <option value="yes">Yes, I do have my own configuration file</option>
+            </param>
+            <when value="yes">
```

Comment:
Just saying most of the label and help text I do not understand. Is it expected that this tool is only used by ODV experts?

Reply:
Yes pretty much, We need to improve a bit the labels and the help but this not for beginners and for ODV experts